src/App.jsx

```diff
@@ -1,16 +1,25 @@
-import ToDoList from './ToDoList.jsx';
-import TodoForm from './TodoForm.jsx';
 import { useState } from 'react';
+import ToDoList from './ToDoList';
+import TodoForm from './TodoForm';
 
 function App() {
-  const [newTodo, setnewTodo] = useState('example text');
+  const [todoList, setTodoList] = useState([]);
+
+  const addTodo = (title) => {
+    console.log('todo => ', title);
```

Comment:
It is not ideal to push your code with console.log. I will suggest to remember to remove them as we consider your main as the production